libqtile/core/manager.py

```diff
@@ -236,7 +237,10 @@ def gobject_thread():
             await self.finalize()
 
     def loop(self):
-        self._eventloop.run_until_complete(self.async_loop())
+        try:
+            self._eventloop.run_until_complete(self.async_loop())
+        except CancelledError:
+            pass
```

Comment:
Why this diff?

Reply:
See the SO from the commit message; I guess the expected behavior is that it throws a CancelledError.